generator/ServiceClientGeneratorLib/Customizations.cs

```diff
@@ -600,6 +600,22 @@ public List<string> NoArgOverloads
             }
         }
 
+        /// <summary>
+        /// For Lists of enums, the generator automatically treats the enum as a string via passing in true to "treatEnumAsString".
+        /// This customization overrides that so that the enum is treated as an enum. 
+        ///     "treatEnumsAsEnums":{
+        ///        "ObjectAttributesList": true
+        /// }
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public bool OverrideTreatEnumsAsString(string shapeName)
+        {
+            var data = _documentRoot[OverrideTreatEnumsAsStringKey];
+            if (data == null) return false;
+            return data[shapeName]  == null ? false : true;
```

Comment:
Since the objects in the list have a value of `true` or `false` whether we should treat enums as strings we should check the value here and not just whether it exists.